tensorflow_addons/optimizers/weight_decay_optimizers_test.py

```diff
@@ -27,6 +27,7 @@
 WEIGHT_DECAY = 0.01
 
 
+@test_utils.run_all_in_graph_and_eager_modes
```

Comment:
Don't think this is necessary. The expected use case would be to extend `OptimizerTestBase`, and wrap the child class with `test_utils.run_all_in_graph_and_eagar_modes`. 

What do you think @seanpmorgan?

Reply:
Thanks @Squadrick. I updated it at child class.